tests/TestDocs.hs

```diff
@@ -29,13 +30,15 @@ publishOpts = Publish.defaultPublishOptions
   where testVersion = ("v999.0.0", Version [999,0,0] [])
 
 main :: IO ()
-main = do
-  pushd "examples/docs" $ do
-    Docs.Package{..} <- Publish.unsafePreparePackage publishOpts
-    forM_ testCases $ \(P.moduleNameFromString -> mn, pragmas) ->
-      let mdl = takeJust ("module not found in docs: " ++ P.runModuleName mn)
-                         (find ((==) mn . Docs.modName) pkgModules)
-      in forM_ pragmas (flip runAssertionIO mdl)
+main = pushd "examples/docs" $ do
+  res <- Publish.preparePackage publishOpts
+  case res of
+    Left e -> preparePackageError e
```

Comment:
I don't think it would be appropriate to print a message about git submodules here. Everything under examples/docs is just a normal file, a part of this repo, isn't it?